### PR TITLE
Remove opal collid filters in batch harvester and harvester error report

### PR DESCRIPTION
**Pull request by kyule**

### CODE REVIEW

**Review by sunray1 — APPROVED** [review]

File: neon/classes/OccurrenceHarvester.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function getHarvestReport($shipmentPK){
 		$retArr = array();
 		$sql = 'SELECT s.errorMessage AS errMsg, COUNT(s.samplePK) as sampleCnt, COUNT(o.occid) as occurrenceCnt '.
 			'FROM NeonSample s LEFT JOIN omoccurrences o ON s.occid = o.occid '.
-			'WHERE s.checkinuid IS NOT NULL AND s.sampleReceived = 1 AND (o.collid NOT IN(81,84) OR o.collid IS NULL) ';
+			'WHERE s.checkinuid IS NOT NULL AND s.sampleReceived = 1';
```
Comment (sunray1) [inline neon/classes/OccurrenceHarvester.php:43]:
```suggestion
			'WHERE s.checkinuid IS NOT NULL AND s.sampleReceived = 1 AND o.collid IS NULL';
```

**Review by sunray1 — CHANGES_REQUESTED** [review]

File: neon/classes/OccurrenceHarvester.php (hunk 1 above)
Comment (sunray1) [reply neon/classes/OccurrenceHarvester.php:43]: Haven't tested this or anything, just noticed it was in the original code

**Review by sunray1 — APPROVED** [review]

**Comment by sunray1** [comment]
@kyule sorry you were right the first time, the null filter shouldn't be in there. My bad!

**Comment by kyule** [comment]
@sunray1 It's really fine either way. It's just whether you want to include ones that have been harvested previously or not in that report. TBH I wouldn't mind just axing this particular report altogether (minus the occurrences not harvested bit at the very top)-- you can see all the errors on the harvesterreports page and it makes the batch harvester page take a long time to load.

**Pull request merged**